app/src/test/java/com/courseproject/tindar/usecases/likelist/LikeListInteractorUnitTest.java

```diff
@@ -0,0 +1,111 @@
+package com.courseproject.tindar.usecases.likelist;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+
+public class LikeListInteractorUnitTest {
+    /** This class tests the implementation and return values of LikeListInteractor **/
+
+    private static final String USER_ID_1 = "user1";
+    private static final String USER_ID_2 = "user2";
+
+    private static class MockLikeListDsGateway implements LikeListDsGateway {
```

Comment:
Can I confirm that you wrote this only in Test because the plan is for the actual implementation to come in a different PR?

Reply:
Mock test because I am not connected to database in my PR, also easier to test with clean mock data

Reply:
Who's in charge of connecting it to the database?

Reply:
Forgot to mention: it would probably be good to document the preconditions and postconditions for the methods somewhere. E.g. assuming that both users are in the database and making it an issue for later, returning an error message if the either user isn't in the database.